xml/admin_gui_dashboard.xml

```diff
@@ -653,16 +666,17 @@
   <sect2 xml:id="dashboard-editing-roles">
    <title>Editing Custom Roles</title>
    <para>
-    Click a custom role's table row and then select <guimenu>Edit</guimenu> in
+    Click a user's table row to highlight selection. Select <guimenu>Edit</guimenu> in
     the top left of the table heading to edit a description and permissions of
-    the custom role. Confirm with <guimenu>Update Role</guimenu>.
+    the custom role. Confirm with <guimenu>Edit Role</guimenu>.
    </para>
   </sect2>
 
   <sect2 xml:id="dashboard-deleting-roles">
    <title>Deleting Custom Roles</title>
    <para>
-    Click a role's table row and then select <guimenu>Delete</guimenu> to
+    Click a role's table row to highlight selection. Select the drop-down button
```

Comment:
And again. You're consistent, I have to give you that. It still reads weirdly to me, though.